CHANGELOG.md

```diff
@@ -25,6 +25,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   any changes to the database. The exception includes an error code and an optional 
   description which the framework saves to the storage for later retrieval. (#392)
 - `ListProofValidator` returns an instance of `NavigableMap` instead of `Map`. (#457)
+- `exonum-java-proofs` module becomes `exonum-java-binding-common` module 
+  with `com.exonum.binding.common` root package. (#459)
+- Refactor`crypto` package to `exonum-java-binding-common` module. (#467)
```

Comment:
Shan't we go here with a hierarchical lists:
* `exonum-java-proofs` module becomes `exonum-java-binding-common` module:
  * Move `X` to `Y`